### ISSUE TITLE
az interactive not working after update to latest az cli

### ISSUE THREAD

**Issue by danvanderboom** [Azure/azure-cli#16931]
## Describe the bug

**Command Name**
`az interactive`

**Errors:**
```
'azext_interactive'
Traceback (most recent call last):
Local\Temp\pip-unpacked-wheel-yccaebu4\knack\cli.py, ln 233, in invoke
azure\cli\core\commands\__init__.py, ln 664, in execute
azure\cli\core\commands\__init__.py, ln 727, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 720, in _run_job
AppData\Local\Temp\pip-unpacked-wheel-rg200gj8\six.py, ln 703, in reraise
azure\cli\core\commands\__init__.py, ln 698, in _run_job
azure\cli\core\commands\__init__.py, ln 331, in __call__
pip-unpacked-wheel-8u6uejh7\azure\cli\core\__init__.py, ln 816, in default_command_handler
azure\cli\command_modules\interactive\custom.py, ln 27, in start_shell
azure\cli\core\extension\operations.py, ln 479, in reload_extension
pip-unpacked-wheel-8u6uejh7\azure\cli\core\util.py, ln 722, in reload_module
KeyError: 'azext_interactive'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az interactive --update`

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.19.1

Extensions:
interactive 0.4.4
storage-preview 0.7.0

```

**Comment by danvanderboom** [comment]
C:\Source>az interactive
This command is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Run `az interactive --update` and try again.

C:\Source>az interactive --update
This command is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Updating the Interactive extension to the latest available...
The command failed with an unexpected error. Here is the traceback:
'azext_interactive'
Traceback (most recent call last):
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-yccaebu4\knack\cli.py", line 233, in invoke
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\commands\__init__.py", line 664, in execute
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\commands\__init__.py", line 727, in _run_jobs_serially
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\commands\__init__.py", line 720, in _run_job
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-rg200gj8\six.py", line 703, in reraise
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\commands\__init__.py", line 698, in _run_job
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\commands\__init__.py", line 331, in __call__
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\__init__.py", line 816, in default_command_handler
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-7as_bwui\azure\cli\command_modules\interactive\custom.py", line 27, in start_shell
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\extension\operations.py", line 479, in reload_extension
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-unpacked-wheel-8u6uejh7\azure\cli\core\util.py", line 722, in reload_module
KeyError: 'azext_interactive'
To open an issue, please run: 'az feedback'

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to check is this related with module index updating? thanks.

**Comment by fengzhou-msft** [comment]
The fix will be released with azure-cli `2.22.0` on 04/13/2021.
